packages/@aws-cdk/aws-iotevents/lib/input.ts

```diff
@@ -11,8 +12,66 @@ export interface IInput extends IResource {
    * @attribute
    */
   readonly inputName: string;
+
+  /**
+   * The ARN of the input
+   * @attribute
+   */
+  readonly inputArn: string;
+
+  /**
+   * Grant the indicated permissions on this input to the given IAM principal (Role/Group/User).
+   *
+   * @param grantee The principal (no-op if undefined)
+   * @param actions The set of actions to allow (i.e. "iotevents:BatchPutMessage")
```

Comment:
These are not sentences, so should start with lowercase. Also, `grantee` cannot be `undefined`:

```suggestion
   * @param grantee the principal
   * @param actions the set of actions to allow (i.e. "iotevents:BatchPutMessage")
```
```